### PR TITLE
Proposal/obb resulting

### PR DESCRIPTION
**Pull request by kafkapre**

### CODE REVIEW

File: proto/ots/ticket_result.proto [hunk 1]
```diff
@@ -94,8 +94,16 @@ message TicketResultState {
   map<string, TicketResultStateBetInfo> bet_info = 4;
 
   // Rejection details of selections.
+  // Key represents Selection ID must has a specific format:
+  // * Regular TicketSelection: "od:<product_id>/<event_id>/<market_id>/<outcome_id>?<specifier1>=<value1>[&<specifierN=valueN>]*"
```
Comment (Radek-Pysny) [inline proto/ots/ticket_result.proto]:
```suggestion
  // * Regular TicketSelection: "od:<product_id>:<event_id>/<market_id>/<outcome_id>?<specifier1>=<value1>[&<specifierN=valueN>]*"
```

File: proto/ots/ticket_result.proto [hunk 2]
```diff
@@ -94,8 +94,16 @@ message TicketResultState {
...
+  // * OBB TicketSelection:  "od:obb_session:xxxx-xxxx-xxxx-xxxx"
```
Comment (Radek-Pysny) [inline proto/ots/ticket_result.proto]:
```suggestion
  // * OBB TicketSelection:  "od:obb_session:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
```

File: proto/ots/ticket_result.proto [hunk 3]
```diff
@@ -94,8 +94,16 @@ message TicketResultState {
...
   map<string, TicketResultStateSelectionInfo> selection_info = 5;
 
+  // Rejection details of BetBuilder (OBB) sub-selections.
+  // Key represents only regular Selection ID and must has a specific format:
+  // "od:<product_id>/<event_id>/<market_id>/<outcome_id>?<specifier1>=<value1>[&<specifierN=valueN>]*"
```
Comment (Radek-Pysny) [inline proto/ots/ticket_result.proto]:
```suggestion
  // "od:<product_id>:<event_id>/<market_id>/<outcome_id>?<specifier1>=<value1>[&<specifierN=valueN>]*"
```

**Review by Radek-Pysny — APPROVED** [review]
I am OK with the design itself. However, there are some comments due to content of comments.

**Review by bednarradek — APPROVED** [review]

**Review by melias122 — APPROVED** [review]

**Pull request merged**